website/templates/util/render_node.mako

```diff
@@ -119,8 +119,8 @@
 % else:
     <li
         node_reference="${summary['id']}:${'node' if summary['primary'] else 'pointer'}"
-        class="project list-group-item list-group-item-node unavailable">
```

Comment:
@haoyuchen1992 That style is the cause of the Trello card. Having background be dark makes it look like a header or footer in that list item group. 

@sloria yes that is intentional. I should have removed the style as well that Haoyu is referring to. I can push that as well if you want. 

As long as unavailable is not being used by js it should be fine, and I don't think it gets a use like that.